src/components/CardArt.vue

```diff
@@ -74,12 +85,15 @@ $default-art-width: 626px;
   display: flex;
   align-items: center;
   justify-content: center;
+  text-align: center;
+  padding: 1rem;
```

Comment:
added some padding here because the class/saga art crops don't give you much space and the text was butting up against the edge.